examples/shader/compute_shader_game_of_life.rs

```diff
@@ -60,10 +61,10 @@ pub struct GameOfLifeComputePlugin;
 
 impl Plugin for GameOfLifeComputePlugin {
     fn build(&self, app: &mut App) {
+        app.add_plugin(ExtractResourcePlugin::<GameOfLifeImage>::default());
```

Comment:
Probably should leave a doc comment explaining what this does so that people viewing the example understand.

Reply:
Hopefully resolved.